mne/decoding/tests/test_base.py

```diff
@@ -20,7 +63,9 @@ def test_get_coef():
     from sklearn.base import TransformerMixin, BaseEstimator
     from sklearn.pipeline import make_pipeline
     from sklearn.preprocessing import StandardScaler
-    from sklearn.linear_model import LinearRegression
+    from sklearn.linear_model import Ridge
+
+    lm = LinearModel(Ridge())
```

Comment:
I thought it would be useful to write down explicitly that the Ridge should get somewhat close but not exactly the result. You think we should just check for the LinearRegression case?